website/docs/docs/cloud/about-cloud/architecture.md

```diff
@@ -3,52 +3,42 @@ title: Architecture
 id: architecture
 ---
 
-This page is intended to help both practitioners as well as those with interest understand the architecture and data flow of the hosted dbt Cloud product.
+This page is intended to help practitioners and those interested in the architecture and data flow of the hosted dbt Cloud product.
 
-### Application Data Flows
+## About dbt Cloud architecture
 
-The dbt Cloud application is comprised of a set of static components, as well as a set of dynamic components. The static components are constantly running to serve highly available dbt Cloud functionality, for example, the dbt Cloud web application. The dynamic components are created just-in-time to fill background jobs or a user request to use the IDE. These components are enumerated below.
+The dbt Cloud application comprises a set of static and dynamic components. The static components are constantly running to serve highly available dbt Cloud functionality, for example, the dbt Cloud web application. The dynamic components are created ad-hoc to fill background jobs or requests to use the IDE. 
 
-<img src="/img/docs/dbt-cloud/on-premises/data-flows.png" />
+dbt Cloud is available in most regions around the world in both [single tenant](/docs/cloud/about-cloud/tenancy#single-tenant)(AWS and Azure) and [multi-tenant](/docs/cloud/about-cloud/tenancy#multi-tenant) configurations.  
+
+dbt Cloud uses PostgreSQL for its backend, S3-compatible Object Storage systems for logs and artifacts, and a Kubernetes storage solution for creating dynamic, persistent volumes. 
 
-#### Static Application Components
+All data at rest on dbt Cloud servers is protected using AES-256 encryption. 
 
-- **api gateway**: The API gateway is the entrypoint for all client requests to dbt Cloud. The api gateway serves static content, and contains logic for routing requests within the dbt Cloud application.
-- **app**: The app is the dbt Cloud application server. It consists of a Django application capable of serving dbt Cloud REST API requests.
-- **scheduler**: The scheduler is a continuously running process that orchestrates background jobs in dbt Cloud. It consists of two components: the scheduler container which provisions dynamic resources just-in-time, and the background cleanup container which performs maintenance tasks on the dbt Cloud database, including flushing logs from dbt runs out into the object store.
+<img src="/img/docs/dbt-cloud/on-premises/data-flows.png" />
 
-#### Dynamic Application Components
+For a more detailed breakdown of the dbt Cloud apps, [download the advanced architecture guide](./assets/dbt-cloud-advanced-architecture-guide.pdf).
```

Comment:
When I click this link I get a 404 but if I right-click and open in new tab it works. Can you ask @JKarlavige and team for their advice?

Reply:
Someone from dev is going over this now

Reply:
Going to use a Google Drive link until this gets figured out